installer/bundle/bundle_skel_Linux.sh

```diff
@@ -330,20 +330,18 @@ shouldInstall_scx()
 
 remove_and_install()
 {
-    check_if_pkg_is_installed apache-cimprov
-    if [ $? -eq 0 ]; then
-        pkg_rm apache-cimprov
-    fi
-    check_if_pkg_is_installed mysql-cimprov
-    if [ $? -eq 0 ]; then
-        pkg_rm mysql-cimprov
-    fi
-    check_if_pkg_is_installed scx
-    if [ $? -eq 0 ]; then
+    if [ -f /opt/microsoft/scx/bin/uninstall ]; then
```

Comment:
I'm unsure if, in actuality, the uninstall script won't exist anymore.

Very old versions of SCX didn't have an uninstall script. So the code at line 336 (with your changes) would only be used for very old versions of SCX. I wonder how long we need to support very old versions of SCX? Older than 2012R2? Did 2012R2 have an uninstall script?

Also, because you're now referring to the uninstall script (which is the correct thing), I went and looked at the uninstall script itself. It needs to be updated to refer to OMI's service_control script for stopping/starting OMI. The OMI service name has changed over the years, and that's the only way to make sure you're referring to the correct thing! Can you pick that up in this change and take care of it?

As per the questions above, you may want to have fallback code if OMI's service_control script doesn't exist. But if it does exist, you should absolutely use it and depend upon it to do the right stuff.